Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py

```diff
@@ -65,7 +65,7 @@ def get_token_request(username, password):
         'NetWitness-Version': VERSION
     }
 
-    response = requests.post(url, headers=get_token_headers, data=username_password, verify=USE_SSL)
+    response = requests.post(url, headers=get_token_headers, data=username_password, verify=USE_SSL, proxies=PROXIES)
```

Comment:
please move the proxy handling to be standard(handle_proxy, without passing argument) if there is no proof that it helps. as this is why it looks like from resent issue updates